### PR TITLE
chore: CXSPA-3794 - SSR contextual logging extensibility

### PR DESCRIPTION
**Pull request by pawelfras**
This PR covers some minor issues with the extensibility of the logic provided for SSR contextual logging. The PR is related to the ID of CXSPA-3794 as the extensibility gaps came out while writing the documentation for this feature.

### CODE REVIEW

File: core-libs/setup/ssr/logger/loggers/default-express-server-logger.ts [hunk 1]
```diff
@@ -20,48 +20,75 @@ import {
...
   }
 
+  /**
+   * Maps a Request object into a JavaScript object with specific properties.
+   *
+   * @protected
+   * @param request - An Express Request object.
+   * @returns - An object with properties "url" and "cxRequest" .
+   * "url" is the original URL from the request and "cxRequest" is an object containing additional information about the request.
+   * "cxRequest" contains properties "uuid" and "timeReceived"
```
Comment (Platonn) [inline core-libs/setup/ssr/logger/loggers/default-express-server-logger.ts]: AFAIK, we don't have anything like "cxRequest", do we?
```suggestion
   * @returns - An object with properties "url" and "request" .
   * "url" is the original URL from the request and "request" is an object containing additional information about the request.
   * "request" contains properties "uuid" and "timeReceived"
```

Comment (pawelfras) [reply core-libs/setup/ssr/logger/loggers/default-express-server-logger.ts]: Oh, yes. Sorry for that. Probably it was a Copilot's interpretation of `...request.res?.locals.cx.request`

Comment (pawelfras) [reply core-libs/setup/ssr/logger/loggers/default-express-server-logger.ts]: Actually, it's worth changing it to `cx.request` to distinguish metadata about the request contained here and `request: Request` that is passed to this method.

Comment (Platonn) [reply core-libs/setup/ssr/logger/loggers/default-express-server-logger.ts]: as discussed offline, it will be all under `request` property

**Review by Platonn — APPROVED** [review]

**Pull request merged**